src/rules/s3/S3BucketServerSideEncryptionEnabled.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
We don't need this rule. It is no longer necessary to check for S3 Server Side Encryption as [S3 now encrypts all objects](https://aws.amazon.com/blogs/aws/amazon-s3-encrypts-new-objects-by-default/) by default. 

The previous rule that checked for S3 SSE [was removed](https://github.com/cdklabs/cdk-nag/issues/1190) due to this new feature.

Reply:
Thats something new I learned today. I knew that the default had been changed, but I had assumed it was still possible to do something like this, but is not. :-)

```
new s3.Bucket(this, 'bucket', {
  encryption: s3.BucketEncryption.UNENCRYPTED
})
```

Resolved by removing rule.